execution/evm/vm.go

```diff
@@ -921,6 +926,10 @@ func (vm *VM) call(caller, callee acm.MutableAccount, code, input []byte, value
 				}
 			}
 
+			if err == ErrExecutionReverted {
```

Comment:
This should be within the block above containing line 914, where the error is checked for readability. Note on develop I have renamed this `err` (which shadows the return value `err`!) to `callErr`, when you rebase against develop (which you need to do) you'll need to change that.

Reply:
You renamed err (in entire case definition for CALL, DELEGATECALL, and CALLCODE) to callErr, or you changed ErrExecutionReverted to callErr? Not sure what to do here other than move the above check to block beginning at LN# 914

Reply:
I will assume you meant you renamed err and not ErrExecutionReverted and push changes